packages/template-retail-react-app/app/hooks/use-update-shopper-context.js

```diff
@@ -17,9 +17,15 @@ import {useQueryClient} from '@tanstack/react-query'
 
 // Hooks
 import useMultiSite from '@salesforce/retail-react-app/app/hooks/use-multi-site'
+import {useShopperContextSearchParams} from '@salesforce/retail-react-app/app/hooks/use-shopper-context-search-params'
 
 // Constants
-import {useShopperContextSearchParams} from '@salesforce/retail-react-app/app/hooks/use-shopper-context-search-params'
+// Add any query keys that need to be refreshed after the Shopper Context is updated
+const QUERIES_TO_INVALIDATE = [
+    '/product-search',
+    'instagram-promo-banner-desktop',
+    'instagram-promo-banner-mobile'
+]
 
```

Comment:
customers can add the query keys they want refetched when the shopper context is updated.

Pro: 
- easy to use: customers can just add the substring for the query key they want refetched here

Con:
- could be bad since this hook is directly accessing the react queries
- if a generic key is put in, queries that the customer didn't want to be refetched might be

<img width="1792" alt="Screenshot 2024-10-09 at 2 32 19 PM" src="https://github.com/user-attachments/assets/b79dafbf-d914-4a3d-a67e-34ef99912cc8">